src/pages/FavouritePage/index.jsx

```diff
@@ -1,7 +1,7 @@
 import React, { useState, useEffect } from "react";
```

Comment:
In the folder name no need to specify the word page since it is inside the page folder.